client/src/components/CityList.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable import/extensions */
+import React from 'react';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import { makeStyles } from '@material-ui/core/styles';
+import styles from '../styles.js';
+
+const useStyles = makeStyles(styles);
+
+const CityList = ({ cities, setCity }) => {
+  const classes = useStyles();
+  // const preventDefault = (event) => event.preventDefault();
+  return (
+    <Box className={classes.cityList}>
+      { cities.map((city) => {
+        const url = `/homes/${city.name}`;
+        return (
+          <Typography key={city._id} className={classes.cities}>
+            <Button href="#" onClick={() => { setCity(city.name); }}>
```

Comment:
Don't have to wrap setCity(city.name) in curly brackets! Can just call it directly after the arrow:

`onClick={() => setCity(city.name)}`